### PR TITLE
EUI-5668

### PR DESCRIPTION
**Pull request by maliumer**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5668

### Change description ###
Welsh language -Disclaimer

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/phase-banner/phase-banner.component.ts [hunk 1]
```diff
@@ -18,7 +19,12 @@ export class PhaseBannerComponent implements OnInit {
   }
 
   public toggleLanguage(lang: RpxLanguage): void {
+    this.noBanner = (lang === 'cy');
     this.langService.language = lang;
   }
 
+  public closeBanner() {
+    this.noBanner = true;
```
Comment (rayliangatos) [inline src/app/components/phase-banner/phase-banner.component.ts:27]: do we need to store this in sessionStorage or localStorage? Could you confirm with Adeel whether need to remember this selection for next time logon?

File: src/cases/containers/case-list/case-list.component.spec.ts [hunk 2]
```diff
@@ -39,6 +40,16 @@ describe('CaseListComponent', () => {
...
+      imports: [
+        RpxTranslationModule.forRoot({
+          baseUrl: '',
+          debounceTimeMs: 300,
+          validity: {
+            days: 1
+          },
+          testMode: true
+        })
+      ],
```
Comment (rayliangatos) [inline src/cases/containers/case-list/case-list.component.spec.ts:52]: do you have to import this with forRoot/forChild for every test?

File: src/work-allocation-2/components/available-tasks-filter/available-tasks-filter.component.spec.ts [hunk 3]
```diff
@@ -38,10 +39,12 @@ describe('AvailableTasksFilterComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [
-        ExuiCommonLibModule
+        ExuiCommonLibModule,
+        RpxTranslationModule.forChild()
```
Comment (rayliangatos) [inline src/work-allocation-2/components/available-tasks-filter/available-tasks-filter.component.spec.ts:43]: do you have to import this with forRoot/forChild for every test?

File: src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.spec.ts [hunk 4]
```diff
@@ -46,10 +47,11 @@ describe('CaseManagerFilterComponent', () => {
         ExuiCommonLibModule,
         HttpClientTestingModule,
         RouterTestingModule,
-        ExuiCommonLibModule
+        RpxTranslationModule.forChild()
```
Comment (rayliangatos) [inline src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.spec.ts:50]: do you have to import this with forRoot/forChild for every test?

File: src/work-allocation-2/containers/all-work-case/all-work-case.component.spec.ts [hunk 5]
```diff
@@ -60,7 +61,15 @@ describe('AllWorkCaseComponent', () => {
...
-        PaginationModule
+        PaginationModule,
+        RpxTranslationModule.forRoot({
+          baseUrl: '',
+          debounceTimeMs: 300,
+          validity: {
+            days: 1
+          },
+          testMode: true
+        })
```
Comment (rayliangatos) [inline src/work-allocation-2/containers/all-work-case/all-work-case.component.spec.ts:72]: do you have to import this with forRoot/forChild for every test?

**Review by rayliangatos — APPROVED** [review]
approved with comments

**Review by SivaprakasamSivaprakasam — APPROVED** [review]

**Pull request merged**